src/storeController/index.ts

```diff
@@ -104,6 +107,15 @@ export default async function (
       }
     }
   }
+
+  async function upload (nodeVersion: string, pkg: {peripheralLocation: string, id: string}) {
```

Comment:
I think a better function signature would be: `upload (builtPkgLocation: string, opts: { pkgId: string, nodeVersion: string })`